Cargo.toml

```
@@ -113,4 +114,4 @@ buildflags = ["--release"]
 polkadot = { path = "/usr/bin/polkadot" }
 
 [package.metadata.rpm.files]
-"../scripts/packaging/polkadot.service" = { path = "/usr/lib/systemd/system/polkadot.service", mode = "644" }
+"../scripts/packaging/polkadot.service" = { path = "/usr/lib/systemd/system/polkadot.service", mode = "644" }
```

```suggestion
"../scripts/packaging/polkadot.service" = { path = "/usr/lib/systemd/system/polkadot.service", mode = "644" }

```

Files should have trailing newlines.